src/main/java/uk/gov/hmcts/reform/waworkflowapi/clients/model/SendMessageRequest.java

```diff
@@ -25,6 +26,7 @@ public SendMessageRequest(@JsonProperty("messageName") String messageName,
         this.processVariables = processVariables;
         this.correlationKeys = correlationKeys;
         this.all = all;
+        this.withoutTenantId = true;
```

Comment:
`true` by default makes sense since we always want to do this.  However, I don't see any tests for this scenario?

Reply:
We should also consider the deployment of the update.  Workflow api should be able to be deployed on it's own and independent of other changes.